### PR TITLE
Remove SB parent from master

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

**Comment by metacosm** [comment]
Damn, editorconfig change took effect and reformated everything making it difficult to see the changes…

**Comment by geoand** [comment]
Shout...

File: pom.xml [hunk 1]
```diff
@@ -1,133 +1,213 @@
...
+  <version>1.5.13-3-SNAPSHOT</version>
+  <packaging>pom</packaging>
+
+  <groupId>io.openshift.booster</groupId>
+  <name>Spring Boot - Istio Distributed Tracing Booster</name>
+  <description>Spring Boot - Istio Distributed Tracing Booster</description>
+
+
+  <properties>
+    <java.version>1.8</java.version>
```
Comment (metacosm) [inline pom.xml]: Outside of the scope of this PR, but this should be removed.

File: pom.xml [hunk 2]
```diff
@@ -1,133 +1,213 @@
...
+
+    <skip.fmp>true</skip.fmp>
```
Comment (metacosm) [inline pom.xml]: Outside of the scope of this PR as well but we need to get to the bottom of this, once and for all.

**Pull request merged**